app/templates/frameworks/updates.html

```diff
@@ -111,7 +111,7 @@
         {% include "toolkit/forms/textbox.html" %}
       {% endwith %}
       <p>
-        The deadline for clarification questions is x
+        The deadline for clarification questions is 5pm BST, 22 September 2015. All responses will be published by 5pm BST, 29 September 2015.
```

Comment:
Can the 'BST' text be wrapped in an `abbr` tag, like we do on the buyers' app?

https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/f19b5a784b136a2a5036e7357de833da3e2afe1a/app/templates/index.html#L49